integration_test/publish.py

```diff
@@ -0,0 +1,17 @@
+import substitution
+from general import *
+
+# clear data
+clear_data()
+
+# install environment
+verify_result(install_environment())
+
+# build adapter image
+verify_result(build_adapter_image(substitution.IMAGEPROJECT))
```

Comment:
We need to pipe in the commit short SHA code here too, sourced from a substitution, similar to before